frontend/public/src/components/EnrolledItemCard.js

```diff
@@ -381,6 +360,13 @@ export class EnrolledItemCard extends React.Component<
                 {certLocation ? (
                   <a href={certLocation}>View certificate</a>
                 ) : null}
+              </div>
+              <br/>
+              <div className="upgrade-item-description">
+                <p>
+                  <strong>Upgrade today</strong> and, upon passing, receive your certificate signed by MIT faculty
+                  to highlight the knowledge and skills you've gained from this MITx course.
+                </p>
```

Comment:
This gets displayed all the time - if I've upgraded to a certificate track course, I probably shouldn't be seeing the upsell text.